README.md

```diff
@@ -40,29 +40,13 @@ make slides
 
 ## Building example bounties
 
-Before building the machine image and testing, you need to compile the bounties binaries.
+Before testing, you need to compile the bounties binaries.
```

Comment:
Will it compile or download?

Reply:
Oops, I meant to say the bounty bundles.
The binaries are now downloaded from `builtins` and embeded.